truss/cli/cli.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import os
 import sys
+import webbrowser
```

Comment:
do we need to add this reuqiment to pyproject.toml? Where's it coming from?

Reply:
part of the python standard lib https://docs.python.org/3/library/webbrowser.html